packages/flutter/test/cupertino/action_sheet_test.dart

```diff
@@ -78,15 +76,15 @@ void main() {
     );
 
     Brightness brightness = Brightness.light;
-    StateSetter stateSetter;
+    StateSetter? stateSetter;
```

Comment:
This could be `late` instead of `?` since you will always assign it before first accessing it.